.svglintrc.mjs

```diff
@@ -298,32 +305,32 @@ export default {
 
           // check if there are some other encoded characters in decimal notation
           // which shouldn't be encoded
-          encodingMatches
-            .filter((m) => !isNaN(m[2]))
-            .forEach((match) => {
-              const decimalNumber = parseInt(match[2]);
-              if (decimalNumber < 128) {
-                _validCodepointsRepr = false;
-
-                const decimalCodepointCharIndex =
-                  getTitleTextIndex(ast.source) + match.index + 1;
-                if (xmlNamedEntitiesCodepoints.includes(decimalNumber)) {
-                  replacement = `"&${
-                    xmlNamedEntities[
-                      xmlNamedEntitiesCodepoints.indexOf(decimalNumber)
-                    ]
-                  };"`;
-                } else {
-                  replacement = String.fromCharCode(decimalNumber);
-                  replacement = replacement == '"' ? `'"'` : `"${replacement}"`;
-                }
+          for (const match of encodingMatches.filter((m) => !isNaN(m[2]))) {
+            const decimalNumber = parseInt(match[2]);
+            if (decimalNumber > 127) {
+              continue;
+            }
+            _validCodepointsRepr = false;
 
-                reporter.error(
-                  `Unnecessary encoded character "${match[0]}" found at index ${decimalCodepointCharIndex}:` +
-                    ` replace it with ${replacement}.`,
-                );
-              }
-            });
+            const decimalCodepointCharIndex =
+              getTitleTextIndex(ast.source) + match.index + 1;
+            if (xmlNamedEntitiesCodepoints.includes(decimalNumber)) {
+              replacement = `"&${
+                xmlNamedEntities[
+                  xmlNamedEntitiesCodepoints.indexOf(decimalNumber)
+                ]
+              };"`;
+            } else {
+              replacement = String.fromCharCode(decimalNumber);
```

Comment:
```suggestion
              replacement = String.fromCodePoint(decimalNumber);
```

Minor: Unicode is better supported in [`String#codePointAt`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/codePointAt) [`String.fromCodePoint()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/fromCodePoint#compared_to_fromcharcode).

Here is [the difference between `String.fromCodePoint()` and `String.fromCharCode()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/fromCodePoint#compared_to_fromcharcode).